### PR TITLE
[16.0] [MIG] delivery_roulier_option

### PR DESCRIPTION
**Pull request by chafique-delli**
@florian-dacosta , @hparfr

### CODE REVIEW

File: delivery_roulier_option/models/stock_quant_package.py [hunk 1]
```diff
@@ -69,9 +69,9 @@ def _roulier_get_customs(self, picking):
                 raise UserError(
                     _(
                         "No H.S. Code on product '%s' nor on it's "
-                        "product category '%s'."
+                        "product category '%(product.display_name, "
+                        "product.categ_id.display_name)s'."
                     )
-                    % (product.display_name, product.categ_id.display_name)
```
Comment (hparfr) [inline delivery_roulier_option/models/stock_quant_package.py]: are you sure about this change ?

**Review by hparfr — CHANGES_REQUESTED** [review]

**Review by hparfr — APPROVED** [review]

**Review by florian-dacosta — APPROVED** [review]

**Pull request merged**